build/run-wasm-rego-tests.sh

```diff
@@ -16,13 +16,18 @@ ASSETS=${ASSETS:-"$PWD/test/wasm/assets"}
 VERBOSE=${VERBOSE:-"0"}
 TESTGEN_CONTAINER_NAME="opa-wasm-testgen-container"
 TESTRUN_CONTAINER_NAME="opa-wasm-testrun-container"
+WASM_BUILD_ONLY=${WASM_BUILD_ONLY:-"false"}
```

Comment:
[nit] I think if we're checking for `!= "true"` below, leaving the default unset would be OK. But it doesn't matter, really.